vagrant/provisioning/roles/haproxy/tasks/main.yml

```diff
@@ -407,6 +407,40 @@
         http-request set-path %[path,regsub(^/{{ arkcase_portal_context | default('portal') }}/,/sitemap/)]
   when: enable_sitemap_config | default(false)
 
+### Portal Robots.txt proxy rules
+- name: ArkCase Portal Robots.txt setup
+  become: true
+  ansible.builtin.lineinfile:
+    backup: true
+    path: "{{ path_haproxy_cfg }}"
+    insertafter: "  acl acl_robots path_beg /robots.txt"
```

Comment:
Did you reverse insertafter and line?  It seems like you want acl_robots after acl_sitemap, not the other way around.